package.json

```diff
@@ -28,6 +28,7 @@
     "ganache-core": "^2.2.1",
     "request": "^2.88.0",
     "request-promise": "^4.2.2",
+    "truffle": "^4.1.14",
```

Comment:
Lets add this to the state channel repo, then we can remove it from here.

Reply:
sure